### PR TITLE
Api 18884 add doc ref

### PR DESCRIPTION
**Pull request by KevinCranmer**
https://vajira.max.gov/browse/API-18884

### CODE REVIEW

File: minimart-manager/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <properties>
     <apache-csv.version>1.8</apache-csv.version>
     <apache-poi.version>5.2.2</apache-poi.version>
-    <data-query.version>5.0.80</data-query.version>
+    <data-query.version>5.0.81-SNAPSHOT</data-query.version>
```
Comment (KevinCranmer) [inline minimart-manager/pom.xml:16]: Will be updated once https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/979 is merged

**Comment by jblefkove-va** [comment]
Did we run `IdValidationAugment`?

`An exception occurred while executing the Java class. java.lang.IllegalStateException: Did not find unique file for ..\datamart\dm-records-43000199\dmDocRef800025983573T.json reference DatamartReference(type=Optional[DocumentReference], reference=Optional[800025983564:T], display=Optional.empty)`

**Comment by KevinCranmer** [comment]
> Did we run `IdValidationAugment`?
> 
> `An exception occurred while executing the Java class. java.lang.IllegalStateException: Did not find unique file for ..\datamart\dm-records-43000199\dmDocRef800025983573T.json reference DatamartReference(type=Optional[DocumentReference], reference=Optional[800025983564:T], display=Optional.empty)`

Ah... I was running `run_synthetic_augment IdValidationAugment` instead of `run_synthetic_augment IdValidationAugment.class`. Looking into it now

**Comment by KevinCranmer** [comment]
> > Did we run `IdValidationAugment`?
> > `An exception occurred while executing the Java class. java.lang.IllegalStateException: Did not find unique file for ..\datamart\dm-records-43000199\dmDocRef800025983573T.json reference DatamartReference(type=Optional[DocumentReference], reference=Optional[800025983564:T], display=Optional.empty)`
> 
> Ah... I was running `run_synthetic_augment IdValidationAugment` instead of `run_synthetic_augment IdValidationAugment.class`. Looking into it now

Wait that's not how you run it either...

**Comment by jblefkove-va** [comment]
ID validation looks good!

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**